.travis.yml

```diff
@@ -83,6 +83,9 @@ before_install:
     # Get our own Swagger verifier
     - wget https://raw.githubusercontent.com/mendersoftware/autodocs/master/verify_docs.py
 
+    # Clone integration repo. for api testing
+    - git clone https://${GITHUB_TOKEN}@github.com/mendersoftware/integration.git
```

Comment:
`GITHUB_TOKEN` will not be set on pull request builds which may mess up those builds a bit.

Why using tokens anyway? that repo is public.